minutes/2018-10-12-node-interactive.md

```diff
@@ -0,0 +1,199 @@
+# Node+JS Interactive 2018 Tooling Group Session
+
+First official meeting of the tooling working group, held as part of the 
+collaborator summit at Node.js interactive.
+
+Schedule next meting [here](https://github.com/nodejs/tooling/issues/1).
+
+## Attendance
+
+### Chris Hiller (@boneskull):
+
+* works for IBM.
+* maintains Mocha.
+* conceived of tooling working group as way to better serve Node users 
+  writing Command Line Tools; taking into account user feedback.
+  
+### Ben Coe (@bcoe)
+
+* PM at npm, Inc.
+* maintains yargs, Istanbul, standard-version.
+* has been working with Christopher and others to beter establish tooling
+  working group.
+  
+### Ian Sutherland (@iansu):
+
+* maintains create-react-app.
+* interested in getting involved in tooling working group.
+
+### Guy Bedford (@guybedord)
```

Comment:
this at-tag is misspelled, should be `@guybedford` (new macbook keybaord[*sic*] ftw)

Reply:
fixed